packages/core/admin/ee/server/routes/features-routes.js

```diff
@@ -43,4 +43,18 @@ module.exports = {
       },
     },
   ],
+  'review-workflows': [
```

Comment:
You'll need to add the new middleware that checks that a feature is available to those routes, most likely.

Reply:
When will we have to do this? Is it already available in main?

Reply:
Indeed, nice catch :pray:

Reply:
Thanks !

Reply:
It should be in main since the 4.6.0 release (Wednesday)

Reply:
what is the purpose of `enableFeatureMiddleware` ? 👀 There are docs about that?

Reply:
I'll add some docs soon, but it returns a 404 if the associated feature is not currently available in the project.

Reply:
makes sense! thank your :)